docs/UserGuide.md

```diff
@@ -127,7 +127,7 @@ specific key-words. Thus, there are **five** types of commands:
 
 ### General
 
-#### Viewing help: `help`
+#### Viewing help : `help`
```

Comment:
This is a formatting error. Most other headings uses a space before hand. AB3's UG contained a mix of space and no space, mostly space. Im converting them to all with space.

Reply:
ah i see. i do think that using a colon here still looks weird. perhaps we could change them to hyphens instead? that way, it would also be consistent with the table of contents.

Reply:
I agree that it is a formatting error, and looks neater and more consistent with space before the colon.